CHANGELOG.md

```diff
@@ -1,6 +1,44 @@
 Changes by Version
 ==================
 
+1.21.0 (2020-11-17)
+-------------------
+
+* Regenerate self-provisioned ES TLS cert when it's outdated ([#1301](https://github.com/jaegertracing/jaeger-operator/pull/1301), [@kevinearls](https://github.com/kevinearls))
+* Update the TestCreateElasticsearchCR unit test to include a case with… ([#1299](https://github.com/jaegertracing/jaeger-operator/pull/1299), [@kevinearls](https://github.com/kevinearls))
+* Enable tolerations support in elasticsearch config ([#1296](https://github.com/jaegertracing/jaeger-operator/pull/1296), [@kevinearls](https://github.com/kevinearls))
+* Update github.com/miekg/dns to v1.1.35 ([#1298](https://github.com/jaegertracing/jaeger-operator/pull/1298), [@objectiser](https://github.com/objectiser))
+* Add serviceType for the collector service ([#1286](https://github.com/jaegertracing/jaeger-operator/pull/1286), [@sschne](https://github.com/sschne))
+* Add env var JAEGER_DISABLED ([#1285](https://github.com/jaegertracing/jaeger-operator/pull/1285), [@sschne](https://github.com/sschne))
+* Fix secret creation when using self provisioned elasticsearch instances ([#1288](https://github.com/jaegertracing/jaeger-operator/pull/1288), [@kevinearls](https://github.com/kevinearls))
+* Remove unused start-openshift.sh script ([#1293](https://github.com/jaegertracing/jaeger-operator/pull/1293), [@objectiser](https://github.com/objectiser))
+* Convert storage type to typed string ([#1282](https://github.com/jaegertracing/jaeger-operator/pull/1282), [@SezalAgrawal](https://github.com/SezalAgrawal))
+* Use New Admin Port Flag ([#1281](https://github.com/jaegertracing/jaeger-operator/pull/1281), [@johanavril](https://github.com/johanavril))
+* Skip generator test until issue 1278 if resolved ([#1279](https://github.com/jaegertracing/jaeger-operator/pull/1279), [@kevinearls](https://github.com/kevinearls))
+* Update instances status using client.Status().update interface ([#1253](https://github.com/jaegertracing/jaeger-operator/pull/1253), [@rubenvp8510](https://github.com/rubenvp8510))
+* Update openshift only tests with new location of examples ([#1274](https://github.com/jaegertracing/jaeger-operator/pull/1274), [@kevinearls](https://github.com/kevinearls))
+* Remove gRPC host-port from being added to the CR (agent) ([#1272](https://github.com/jaegertracing/jaeger-operator/pull/1272), [@jpkrohling](https://github.com/jpkrohling))
+* Sync OTEL config volume/mount and args ([#1268](https://github.com/jaegertracing/jaeger-operator/pull/1268), [@jpkrohling](https://github.com/jpkrohling))
+* Fix csv generation in release script ([#1247](https://github.com/jaegertracing/jaeger-operator/pull/1247), [@rubenvp8510](https://github.com/rubenvp8510))
+* Publish container - dockerx should not use tag BUILD_IMAGE ([#1270](https://github.com/jaegertracing/jaeger-operator/pull/1270), [@morlay](https://github.com/morlay))
```

Comment:
Even though this (and the next) are related to internal builds, this user has provided useful contributions, so should be recognized in the changelog.

Reply:
+1